frontend/packages/ocs-independent-plugin/src/models.ts

```diff
@@ -0,0 +1,14 @@
+import { K8sKind } from '@console/internal/module/k8s';
+
+export const OCSServiceModel: K8sKind = {
```

Comment:
There is already an `OCSServiceModel` in `packages/ceph-storage-plugin/src/models.ts`.

Re-defining it here will cause the corresponding extension check (*) to fail.

(*) `packages/console-app/src/__tests__/extension-checks/models.spec.ts`

The need to re-define a model also indicates a close relationship between Ceph & OCS/independent plugins. Maybe put the code into existing Ceph plugin?